flowrs-build-designer/rete/flowrs/contextCreator.ts

```diff
@@ -1,294 +1,310 @@
-import {
-    type FlowNode,
-    type FlowConnection,
-    type FlowProject,
-    type ProjectIdentifier
-} from "~/repository/modules/projects";
-import {FlowrsNode} from "~/rete/flowrs/flowrsNode";
-import {NodeEditor} from "rete";
-import {type ItemDefinition} from "rete-context-menu-plugin/_types/presets/classic/types";
-import {useProjectsStore} from "~/store/projectStore";
-import {ContextMenuPlugin, Presets as ContextMenuPresets} from "rete-context-menu-plugin";
-import type {Schemes} from "~/rete/flowrs/editor";
-import {Connection} from "~/rete/flowrs/editor";
-import type {TypeDefinition} from "~/repository/modules/packages";
-import {navigateTo} from "#app";
-
-
+import { type FlowNode, type FlowConnection, type FlowProject, type ProjectIdentifier } from "~/repository/modules/projects";
+import { FlowrsNode } from "~/rete/flowrs/flowrsNode";
+import { NodeEditor } from "rete";
+import { type ItemDefinition } from "rete-context-menu-plugin/_types/presets/classic/types";
+import { useProjectsStore } from "~/store/projectStore";
+import { ContextMenuPlugin, Presets as ContextMenuPresets } from "rete-context-menu-plugin";
+import type { Schemes } from "~/rete/flowrs/editor";
+import { Connection } from "~/rete/flowrs/editor";
+import type { TypeDefinition } from "~/repository/modules/packages";
+import { usePackagesStore } from "~/store/packageStore.js";
+import { navigateTo } from "#app";
+
+const packagesStore = usePackagesStore();
```

Comment:
das hier oben könnte man in die methoden ziehen wos benutzt wird

sonst schwirrt so ne const ohne klasse rum ^^

Reply:
Erledigt.